src/api/billing.ts

```diff
@@ -48,7 +74,41 @@ export const setTenantPrimaryPaymentMethod = (tenant: string, id: string) => {
     });
 };
 
-interface InvoiceLineItem {
+export interface ManualBill {
+    usd_cents: number;
+    description: string;
+    date_start: string;
+    date_end: string;
+    tenant?: string;
+}
+
+const manualBillsQuery = [
+    'description',
+    'usd_cents',
+    'date_start',
+    'date_end',
+].join(', ');
+
+export const getManualBills = (
+    billed_prefix: string,
+    active_during: string | Date
+): PromiseLike<PostgrestResponse<ManualBill>> => {
+    const fmt = "yyyy-MM-dd' 00:00:00+00'";
```

Comment:
This format is in three spots for billing. Please update this so it is shared in a single spot. `billing-utils` is probably a good spot.